.github/workflows/e2e.yaml

```
@@ -0,0 +1,50 @@
+name: E2E over minikube
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  e2eTests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ## we could run priviledgeMode this in serie instead of in //
+        ## PRO less total time, less expensive CONTRA slower to get results
+        priviledgeMode: [ "normal", "unprivileged" ]
+        k8sVersion: [ "v1.15.7", "v1.16.7", "v1.17.9", "v1.18.6", "v1.19.2" ]
```

Latest patch versions are:

```suggestion
        k8sVersion: [ "v1.15.12", "v1.16.15", "v1.17.14", "v1.18.12", "v1.19.4" ]
```
There is probably no reason to update, but I don't see any reason for not doing it either :)
As usual my motivation is to keep everything as latest as possible since likely we won't touch these on a while.